[icu] build failure

**Host Environment**
- OS: Windows (Github action virtual environment?)
- Compiler: unkown. cl.exe?

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install qscintilla`

Automake seems to be a dependency. Dependencies before `automake` install and build fine.

***EDIT***: Automake 1.15 source is not available for download on msys2.org anymore. Oldest version is not automate 1.16 - thus the download fails.

**Failure logs** 
```
-- Downloading https://repo.msys2.org/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:181 (message):
      
      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "***your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  scripts/cmake/vcpkg_acquire_msys.cmake:97 (vcpkg_download_distfile)
  scripts/cmake/vcpkg_acquire_msys.cmake:232 (msys_package)
  ports/icu/portfile.cmake:80 (vcpkg_acquire_msys)
  scripts/ports.cmake:79 (include)


Error: Building package icu:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: icu:x86-windows
  Vcpkg version: 2020.06.15-nohash
```

Full log:

```
Computing installation plan...
The following packages will be built and installed:
  * atlmfc[core]:x86-windows
  * brotli[core]:x86-windows
  * bzip2[core]:x86-windows
  * double-conversion[core]:x86-windows
  * egl-registry[core]:x86-windows
  * freeglut[core]:x86-windows
  * freetype[bzip2,core,png]:x86-windows
  * harfbuzz[core]:x86-windows
  * icu[core]:x86-windows
  * jasper[core]:x86-windows
  * libjpeg-turbo[core]:x86-windows
  * liblzma[core]:x86-windows
  * libpng[core]:x86-windows
  * libpq[core,openssl,zlib]:x86-windows
  * libwebp[core,nearlossless,simd,unicode]:x86-windows
  * opengl[core]:x86-windows
  * openssl[core]:x86-windows
  * openssl-windows[core]:x86-windows
  * pcre2[core]:x86-windows
    qscintilla[core]:x86-windows
  * qt5-base[core]:x86-windows
  * qt5-declarative[core]:x86-windows
  * qt5-imageformats[core]:x86-windows
  * qt5-multimedia[core]:x86-windows
  * qt5-svg[core]:x86-windows
  * qt5-winextras[core]:x86-windows
  * ragel[core]:x86-windows
  * sqlite3[core]:x86-windows
  * tiff[core]:x86-windows
  * zlib[core]:x86-windows
  * zstd[core]:x86-windows
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x86-windows...
A suitable version of powershell-core was not found (required v7.0.3). Downloading portable powershell-core v7.0.3...
Downloading powershell-core...
  https://github.com/PowerShell/PowerShell/releases/download/v7.0.3/PowerShell-7.0.3-win-x86.zip -> C:\vcpkg\downloads\PowerShell-7.0.3-win-x86.zip
Extracting powershell-core...
A suitable version of 7zip was not found (required v18.1.0). Downloading portable 7zip v18.1.0...
Downloading 7zip...
  https://www.nuget.org/api/v2/package/7-Zip.CommandLine/18.1.0 -> C:\vcpkg\downloads\7-zip.commandline.18.1.0.nupkg
Extracting 7zip...
Starting package 1/31: atlmfc:x86-windows
Building package atlmfc[core]:x86-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\b8\b804bc2b084f60b90e07308743f838249adccb31.zip
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\b8\b804bc2b084f60b90e07308743f838249adccb31.zip
Building package atlmfc[core]:x86-windows... done
Installing package atlmfc[core]:x86-windows...
Installing package atlmfc[core]:x86-windows... done
Elapsed time for package atlmfc:x86-windows: 188.6 ms
Starting package 2/31: brotli:x86-windows
Building package brotli[core]:x86-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\ec\ecb1c8779724df939c46d6165bae660d2f832165.zip
-- Downloading https://github.com/google/brotli/archive/e61745a6b7add50d380cfd7d3883dd6c62fc2c71.tar.gz...
-- Extracting source C:/vcpkg/downloads/google-brotli-e61745a6b7add50d380cfd7d3883dd6c62fc2c71.tar.gz
-- Applying patch install.patch
-- Applying patch fix-arm-uwp.patch
-- Applying patch pkgconfig.patch
-- Using source at C:/vcpkg/buildtrees/brotli/src/6c62fc2c71-a8c4ea9278.clean
-- Configuring x86-windows
-- Building x86-windows-dbg
-- Building x86-windows-rel
-- Downloading https://repo.msys2.org/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-1-any.pkg.tar.xz...
-- Downloading https://repo.msys2.org/mingw/i686/mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst...
-- Fixing pkgconfig file: C:/vcpkg/packages/brotli_x86-windows/lib/pkgconfig/libbrotlicommon.pc
-- Fixing pkgconfig file: C:/vcpkg/packages/brotli_x86-windows/lib/pkgconfig/libbrotlidec.pc
-- Fixing pkgconfig file: C:/vcpkg/packages/brotli_x86-windows/lib/pkgconfig/libbrotlienc.pc
-- Fixing pkgconfig file: C:/vcpkg/packages/brotli_x86-windows/debug/lib/pkgconfig/libbrotlicommon.pc
-- Fixing pkgconfig file: C:/vcpkg/packages/brotli_x86-windows/debug/lib/pkgconfig/libbrotlidec.pc
-- Fixing pkgconfig file: C:/vcpkg/packages/brotli_x86-windows/debug/lib/pkgconfig/libbrotlienc.pc
-- Installing: C:/vcpkg/packages/brotli_x86-windows/share/brotli/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\ec\ecb1c8779724df939c46d6165bae660d2f832165.zip
Building package brotli[core]:x86-windows... done
Installing package brotli[core]:x86-windows...
Installing package brotli[core]:x86-windows... done
Elapsed time for package brotli:x86-windows: 39.42 s
Starting package 3/31: bzip2:x86-windows
Building package bzip2[core]:x86-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\65\6532da60a30562cc34425daa524b71bed22ba786.zip
-- Fetching https://sourceware.org/git/bzip2.git...
-- Extracting source C:/vcpkg/downloads/bzip2-75a94bea3918e612b879d6a11ca64b8689526147.tar.gz
-- Applying patch fix-import-export-macros.patch
-- Using source at C:/vcpkg/buildtrees/bzip2/src/8689526147-1ef169fbaa.clean
-- Configuring x86-windows
-- Building x86-windows-dbg
-- Building x86-windows-rel
-- Fixing pkgconfig file: C:/vcpkg/packages/bzip2_x86-windows/lib/pkgconfig/bzip2.pc
-- Fixing pkgconfig file: C:/vcpkg/packages/bzip2_x86-windows/debug/lib/pkgconfig/bzip2.pc
-- Installing: C:/vcpkg/packages/bzip2_x86-windows/share/bzip2/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\65\6532da60a30562cc34425daa524b71bed22ba786.zip
Building package bzip2[core]:x86-windows... done
Installing package bzip2[core]:x86-windows...
Installing package bzip2[core]:x86-windows... done
Elapsed time for package bzip2:x86-windows: 11.49 s
Starting package 4/31: double-conversion:x86-windows
Building package double-conversion[core]:x86-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\64\64919f420cc656adab2e83b92b857a38a4d6ae7e.zip
-- Note: double-conversion only supports static library linkage. Building static library.
-- Downloading https://github.com/google/double-conversion/archive/v3.1.5.tar.gz...
-- Extracting source C:/vcpkg/downloads/google-double-conversion-v3.1.5.tar.gz
-- Using source at C:/vcpkg/buildtrees/double-conversion/src/v3.1.5-d04c588b2e.clean
-- Configuring x86-windows
-- Building x86-windows-dbg
-- Building x86-windows-rel
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\64\64919f420cc656adab2e83b92b857a38a4d6ae7e.zip
Building package double-conversion[core]:x86-windows... done
Installing package double-conversion[core]:x86-windows...
Installing package double-conversion[core]:x86-windows... done
Elapsed time for package double-conversion:x86-windows: 8.022 s
Starting package 5/31: egl-registry:x86-windows
Building package egl-registry[core]:x86-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\46\46249cce8b6d064f788554df32c7227b1eaa8324.zip
-- Downloading https://github.com/KhronosGroup/EGL-Registry/archive/11478904448bbdf5757b798c856a525aa2b351b1.tar.gz...
-- Extracting source C:/vcpkg/downloads/KhronosGroup-EGL-Registry-11478904448bbdf5757b798c856a525aa2b351b1.tar.gz
-- Using source at C:/vcpkg/buildtrees/egl-registry/src/5aa2b351b1-2a8c3e3154.clean
-- Installing: C:/vcpkg/packages/egl-registry_x86-windows/share/egl-registry/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\46\46249cce8b6d064f788554df32c7227b1eaa8324.zip
Building package egl-registry[core]:x86-windows... done
Installing package egl-registry[core]:x86-windows...
Installing package egl-registry[core]:x86-windows... done
Elapsed time for package egl-registry:x86-windows: 1.271 s
Starting package 6/31: freeglut:x86-windows
Building package freeglut[core]:x86-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\45\451fe7dd9f5baf465327483d736c88091d18e450.zip
-- Trying auto-select mirror...
-- Downloading https://sourceforge.net/projects/freeglut/files/freeglut/3.2.1/freeglut-3.2.1.tar.gz/download...
-- Skipping hash check for C:/vcpkg/downloads/temp/freeglut-3.2.1.tar.gz.
-- The default mirror is in Disaster Recovery mode, trying other mirrors...
-- Trying mirror cfhcable...
-- Using cached C:/vcpkg/downloads/freeglut-3.2.1.tar.gz
-- Skipping hash check for C:/vcpkg/downloads/freeglut-3.2.1.tar.gz.
-- Extracting source C:/vcpkg/downloads/freeglut-3.2.1.tar.gz
-- Applying patch use_targets_to_export_x11_dependency.patch
-- Applying patch macOS_Xquartz.patch
-- Applying patch gcc10.patch
-- Applying patch fix-debug-macro.patch
-- Using source at C:/vcpkg/buildtrees/freeglut/src/aced4bbcd3-d62918a414.clean
-- Configuring x86-windows
-- Building x86-windows-dbg
-- Building x86-windows-rel
-- Installing: C:/vcpkg/packages/freeglut_x86-windows/share/freeglut/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\45\451fe7dd9f5baf465327483d736c88091d18e450.zip
Building package freeglut[core]:x86-windows... done
Installing package freeglut[core]:x86-windows...
Installing package freeglut[core]:x86-windows... done
Elapsed time for package freeglut:x86-windows: 35.69 s
Starting package 7/31: zlib:x86-windows
Building package zlib[core]:x86-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\4f\4fe746e464d74ade82c420fbc5ef7515c83adb51.zip
-- Downloading http://www.zlib.net/zlib-1.2.11.tar.gz...
-- Extracting source C:/vcpkg/downloads/zlib1211.tar.gz
-- Applying patch cmake_dont_build_more_than_needed.patch
-- Applying patch 0001-Prevent-invalid-inclusions-when-HAVE_-is-set-to-0.patch
-- Applying patch add_debug_postfix_on_mingw.patch
-- Using source at C:/vcpkg/buildtrees/zlib/src/1.2.11-63309e48e5.clean
-- Configuring x86-windows
-- Building x86-windows-dbg
-- Building x86-windows-rel
-- Fixing pkgconfig file: C:/vcpkg/packages/zlib_x86-windows/lib/pkgconfig/zlib.pc
-- Fixing pkgconfig file: C:/vcpkg/packages/zlib_x86-windows/debug/lib/pkgconfig/zlib.pc
-- Installing: C:/vcpkg/packages/zlib_x86-windows/share/zlib/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\4f\4fe746e464d74ade82c420fbc5ef7515c83adb51.zip
Building package zlib[core]:x86-windows... done
Installing package zlib[core]:x86-windows...
Installing package zlib[core]:x86-windows... done
Elapsed time for package zlib:x86-windows: 26.52 s
Starting package 8/31: libpng:x86-windows
Building package libpng[core]:x86-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\e6\e6148d6d48d64b806fc24e6425e093ebe1a8bfb1.zip
-- Downloading https://github.com/glennrp/libpng/archive/v1.6.37.tar.gz...
-- Extracting source C:/vcpkg/downloads/glennrp-libpng-v1.6.37.tar.gz
-- Applying patch use_abort.patch
-- Applying patch cmake.patch
-- Applying patch pkgconfig.patch
-- Applying patch pkgconfig.2.patch
-- Using source at C:/vcpkg/buildtrees/libpng/src/v1.6.37-10db9f58e4.clean
-- Configuring x86-windows
-- Building x86-windows-dbg
-- Building x86-windows-rel
-- Fixing pkgconfig file: C:/vcpkg/packages/libpng_x86-windows/lib/pkgconfig/libpng.pc
-- Fixing pkgconfig file: C:/vcpkg/packages/libpng_x86-windows/lib/pkgconfig/libpng16.pc
-- Fixing pkgconfig file: C:/vcpkg/packages/libpng_x86-windows/debug/lib/pkgconfig/libpng.pc
-- Fixing pkgconfig file: C:/vcpkg/packages/libpng_x86-windows/debug/lib/pkgconfig/libpng16.pc
-- Installing: C:/vcpkg/packages/libpng_x86-windows/share/libpng/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\e6\e6148d6d48d64b806fc24e6425e093ebe1a8bfb1.zip
Building package libpng[core]:x86-windows... done
Installing package libpng[core]:x86-windows...
Installing package libpng[core]:x86-windows... done
Elapsed time for package libpng:x86-windows: 14.76 s
Starting package 9/31: freetype:x86-windows
Building package freetype[bzip2,core,png]:x86-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\f1\f188aee839c21a41174167cbf2e53f1a7f505e8c.zip
-- Downloading https://download-mirror.savannah.gnu.org/releases/freetype/freetype-2.10.2.tar.xz...
-- Extracting source C:/vcpkg/downloads/freetype-2.10.2.tar.xz
-- Applying patch 0001-Fix-install-command.patch
-- Applying patch 0003-Fix-UWP.patch
-- Applying patch pkgconfig.patch
-- Applying patch brotli-static.patch
-- Using source at C:/vcpkg/buildtrees/freetype/src/2.10.2-c111da3ccb.clean
-- Configuring x86-windows
-- Building x86-windows-dbg
-- Building x86-windows-rel
-- Fixing pkgconfig file: C:/vcpkg/packages/freetype_x86-windows/lib/pkgconfig/freetype2.pc
-- Fixing pkgconfig file: C:/vcpkg/packages/freetype_x86-windows/debug/lib/pkgconfig/freetype2.pc
-- Installing: C:/vcpkg/packages/freetype_x86-windows/share/freetype/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\f1\f188aee839c21a41174167cbf2e53f1a7f505e8c.zip
Building package freetype[bzip2,core,png]:x86-windows... done
Installing package freetype[bzip2,core,png]:x86-windows...
Installing package freetype[bzip2,core,png]:x86-windows... done
Elapsed time for package freetype:x86-windows: 21.81 s
Starting package 10/31: ragel:x86-windows
Building package ragel[core]:x86-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\2a\2aa5e99091b8e93ca08de750273b4a26ab7a1480.zip
-- Downloading http://www.colm.net/files/ragel/ragel-6.10.tar.gz...
-- Extracting source C:/vcpkg/downloads/ragel-6.10.tar.gz
-- Applying patch 0001-remove-unistd-h.patch
-- Using source at C:/vcpkg/buildtrees/ragel/src/ragel-6-655d7c9ccd.clean
-- Configuring x86-windows
-- Building x86-windows-dbg
-- Building x86-windows-rel
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\2a\2aa5e99091b8e93ca08de750273b4a26ab7a1480.zip
Building package ragel[core]:x86-windows... done
Installing package ragel[core]:x86-windows...
Installing package ragel[core]:x86-windows... done
Elapsed time for package ragel:x86-windows: 1.748 min
Starting package 11/31: harfbuzz:x86-windows
Building package harfbuzz[core]:x86-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\e3\e39f1fc4b031c9394e27bf845b3969898673d3e5.zip
-- Downloading https://github.com/harfbuzz/harfbuzz/archive/2.6.6.tar.gz...
-- Extracting source C:/vcpkg/downloads/harfbuzz-harfbuzz-2.6.6.tar.gz
-- Applying patch 0001-fix-cmake-export.patch
-- Applying patch 0002-fix-uwp-build.patch
-- Applying patch 0003-remove-broken-test.patch
-- Applying patch find-package-freetype-2.patch
-- Applying patch glib-cmake.patch
-- Applying patch fix_include.patch
-- Using source at C:/vcpkg/buildtrees/harfbuzz/src/2.6.6-6db14e87dc.clean
-- Configuring x86-windows
-- Building x86-windows-dbg
-- Building x86-windows-rel
-- Installing: C:/vcpkg/packages/harfbuzz_x86-windows/share/harfbuzz/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\runneradmin\AppData\Local\vcpkg\archives\e3\e39f1fc4b031c9394e27bf845b3969898673d3e5.zip
Building package harfbuzz[core]:x86-windows... done
Installing package harfbuzz[core]:x86-windows...
Installing package harfbuzz[core]:x86-windows... done
Elapsed time for package harfbuzz:x86-windows: 1.634 min
Starting package 12/31: icu:x86-windows
Building package icu[core]:x86-windows...
Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\8c\8c9e3ef9e844e61fe542d8629c275c7e3396c3fd.zip
-- Downloading https://github.com/unicode-org/icu/releases/download/release-67-1/icu4c-67_1-src.tgz...
-- Extracting source C:/vcpkg/downloads/icu4c-67_1-src.tgz
-- Applying patch C:/vcpkg/ports/icu/disable-escapestr-tool.patch
-- Applying patch C:/vcpkg/ports/icu/remove-MD-from-configure.patch
-- Applying patch C:/vcpkg/ports/icu/fix_parallel_build_on_windows.patch
-- Applying patch C:/vcpkg/ports/icu/fix-extra.patch
-- Using source at C:/vcpkg/buildtrees/icu/src/c-67_1-src-39ba50b26e.clean
-- Downloading https://www.python.org/ftp/python/3.8.3/python-3.8.3-embed-win32.zip...
-- Downloading https://repo.msys2.org/msys/x86_64/bash-4.4.023-2-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/diffutils-3.7-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/coreutils-8.32-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/grep-3.0-2-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/sed-4.8-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/libpcre-8.44-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/gawk-5.1.0-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/mpfr-4.1.0-1-x86_64.pkg.tar.zst...
-- Downloading https://repo.msys2.org/msys/x86_64/gmp-6.2.0-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/libreadline-8.0.004-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/ncurses-6.2-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/automake1.15-1.15.1-1-any.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:181 (message):
      
      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "***your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  scripts/cmake/vcpkg_acquire_msys.cmake:97 (vcpkg_download_distfile)
  scripts/cmake/vcpkg_acquire_msys.cmake:232 (msys_package)
  ports/icu/portfile.cmake:80 (vcpkg_acquire_msys)
  scripts/ports.cmake:79 (include)


Error: Building package icu:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: icu:x86-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.
Error: Process completed with exit code 1.
```

**Additional context**
Sorry, but I can't give additional information. I don't have a Windows box to diagnose this issue.

--- a/ports/icu/CONTROL
+++ b/ports/icu/CONTROL
@@ -1,5 +1,6 @@
 Source: icu
-Version: 67.1-3
+Version: 67.1
+Port-Version: 4
 Homepage: http://icu-project.org/apiref/icu4c/
 Description: Mature and widely used Unicode and localization library.
 Supports: !(arm|uwp)

--- a/ports/icu/portfile.cmake
+++ b/ports/icu/portfile.cmake
@@ -77,7 +77,7 @@ else()
     set(CONFIGURE_OPTIONS "${CONFIGURE_OPTIONS} --host=i686-pc-mingw32")
 
     # Acquire tools
-    vcpkg_acquire_msys(MSYS_ROOT PACKAGES make automake1.15)
+    vcpkg_acquire_msys(MSYS_ROOT PACKAGES make automake1.16)
 
     # Insert msys into the path between the compiler toolset and windows system32. This prevents masking of "link.exe" but DOES mask "find.exe".
     string(REPLACE ";$ENV{SystemRoot}\\system32;" ";${MSYS_ROOT}/usr/bin;$ENV{SystemRoot}\\system32;" NEWPATH "$ENV{PATH}")
@@ -85,7 +85,7 @@ else()
     set(ENV{PATH} "${NEWPATH}")
     set(BASH ${MSYS_ROOT}/usr/bin/bash.exe)
 
-    set(AUTOMAKE_DIR ${MSYS_ROOT}/usr/share/automake-1.15)
+    set(AUTOMAKE_DIR ${MSYS_ROOT}/usr/share/automake-1.16)
     file(COPY ${AUTOMAKE_DIR}/config.guess ${AUTOMAKE_DIR}/config.sub DESTINATION ${SOURCE_PATH}/source)
 
     if(VCPKG_CRT_LINKAGE STREQUAL static)

